src/main/studio/kdb/K.java

```diff
@@ -777,11 +799,11 @@ public boolean isNull() {
 
         public String toString(boolean showType) {
             if (isNull())
-                return "0Nd";
+                return "0nd";
```

Comment:
The convention is to use "n" for float and "N" for everything else. q will parse if you input the "n" with the wrong case but it will print the value back using the correct case. Same for w/W.
https://code.kx.com/q/basics/datatypes/

Reply:
Thanks - updated the PR - everything is now upper case except KDouble